### PR TITLE
Style guide fixes

### PR DESCRIPTION
**Pull request by Rick-Anderson**
### Why:

Style guide fixes.  Shorter sentences are easier for native speakers and are far better for MT (Machine Translation).

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/issues/tracking-your-work-with-issues/about-issues.md [hunk 1]
```diff
@@ -19,11 +19,19 @@ topics:
...
 
 ## Quickly create issues
 
-Issues can be created in a variety of ways, so you can choose the most convenient method for your workflow. For example, you can create an issue from a repository,{% ifversion fpt or ghec %} an item in a task list,{% endif %} a note in a project, a comment in an issue or pull request, a specific line of code, or a URL query. You can also create an issue from your platform of choice: through the web UI, {% data variables.product.prodname_desktop %}, {% data variables.product.prodname_cli %}, GraphQL and REST APIs, or {% data variables.product.prodname_mobile %}. For more information, see "[Creating an issue](/issues/tracking-your-work-with-issues/creating-issues/creating-an-issue)."
+Issues can be created in a variety of ways, you can choose the most convenient method for your workflow. For example, you can create an issue from a repository,{% ifversion fpt or ghec %} an item in a task list,{% endif %} a note in a project, a comment in an issue or pull request, a specific line of code, or a URL query. You can  create an issue from your platform of choice:
+
+* Through the web UI,
```
Comment (Rick-Anderson) [inline content/issues/tracking-your-work-with-issues/about-issues.md:28]:
```suggestion
* Through the web UI,
```

File: content/issues/tracking-your-work-with-issues/about-issues.md [hunk 2]
```diff
@@ -19,11 +19,19 @@ topics:
...
```
Comment (Rick-Anderson) [inline content/issues/tracking-your-work-with-issues/about-issues.md:28]:
```suggestion
* Through the web UI
```

File: content/issues/tracking-your-work-with-issues/about-issues.md [hunk 3]
```diff
@@ -19,11 +19,19 @@ topics:
...
+Issues can be created in a variety of ways, you can choose the most convenient method for your workflow. For example, you can create an issue from a repository,{% ifversion fpt or ghec %} an item in a task list,{% endif %} a note in a project, a comment in an issue or pull request, a specific line of code, or a URL query. You can  create an issue from your platform of choice:
+
+* Through the web UI
+* {% data variables.product.prodname_desktop %}
+* {% data variables.product.prodname_cli %}
+* GraphQL and REST APIs
+* {% data variables.product.prodname_mobile %}.
```
Comment (Rick-Anderson) [inline content/issues/tracking-your-work-with-issues/about-issues.md:32]:
```suggestion
* {% data variables.product.prodname_mobile %}
```

**Comment by ramyaparimi** [comment]
@Rick-Anderson 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/issues/tracking-your-work-with-issues/about-issues.md [hunk 4]
```diff
@@ -19,11 +19,19 @@ topics:
 ---
 ## Integrated with GitHub
 
-Issues let you track your work on {% data variables.product.company_short %}, where development happens. When you mention an issue in another issue or pull request, the issue's timeline reflects the cross-reference so that you can keep track of related work. To indicate that work is in progress, you can link an issue to a pull request. When the pull request merges, the linked issue automatically closes.
+Issues let you track your work on {% data variables.product.company_short %}, where development happens. When you mention an issue in another issue or pull request, the issue's timeline reflects the cross-reference so that you can keep track of related work. To indicate that work is in progress, you can link an issue to a pull request. Pull requests addressing an issue typically include a [keyword that automatically closes the issue](issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) when the pull request merges.
```
Comment (Rick-Anderson) [inline content/issues/tracking-your-work-with-issues/about-issues.md:22]:
```suggestion
Issues let you track your work on {% data variables.product.company_short %}, where development happens. When an issue is mentioned from in another issue or pull request, the issue's timeline reflects the cross-reference to make it easier to track the related work. To indicate that work is in progress, you can link an issue to a pull request. Pull requests addressing an issue typically include a [keyword that automatically closes the issue](issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) when the pull request merges.
```

**Comment by ramyaparimi** [comment]
@Rick-Anderson We won’t be accepting this change, as our [style guide ](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#procedural-steps)indicates we start with the initiating action before the result 💛 .

Thanks so much for your time in contributing to the GitHub docs 💖 .

**Comment by Rick-Anderson** [comment]
@ramyaparimi  but the content is incorrect. Mentioning is not enough to close the issue. That's why I added a link.

**Comment by ramyaparimi** [comment]
@Rick-Anderson Thanks so much for reaching out. I will triage this for the team to take a closer look at the changes. Thank you for your patience and passion in contributing to the GitHub docs 💖 .

File: content/issues/tracking-your-work-with-issues/about-issues.md [hunk 5]
```diff
@@ -19,11 +19,19 @@ topics:
...
+Issues can be created in a variety of ways, you can choose the most convenient method for your workflow. For example, you can create an issue from a repository,{% ifversion fpt or ghec %} an item in a task list,{% endif %} a note in a project, a comment in an issue or pull request, a specific line of code, or a URL query. You can  create an issue from your platform of choice:
+
+* Through the web UI
+* {% data variables.product.prodname_desktop %}
+* {% data variables.product.prodname_cli %}
+* GraphQL and REST APIs
+* {% data variables.product.prodname_mobile %}
+
+For more information, see "[Creating an issue](/issues/tracking-your-work-with-issues/creating-issues/creating-an-issue)."
```
Comment (myarb) [inline content/issues/tracking-your-work-with-issues/about-issues.md:34]:
```suggestion
Issues can be created in a variety of ways, so you can choose the most convenient method for your workflow. For example, you can create an issue from a repository,{% ifversion fpt or ghec %} an item in a task list,{% endif %} a note in a project, a comment in an issue or pull request, a specific line of code, or a URL query. You can also create an issue from your platform of choice: through the web UI, {% data variables.product.prodname_desktop %}, {% data variables.product.prodname_cli %}, GraphQL and REST APIs, or {% data variables.product.prodname_mobile %}. For more information, see "[Creating an issue](/issues/tracking-your-work-with-issues/creating-issues/creating-an-issue)."
```

**Review by myarb — DISMISSED** [review]
Thank you for your contribution to the GitHub Docs! We'll merge the addition of a link to the article on keywords shortly.

**Review by ramyaparimi — APPROVED** [review]

**Review by sandrosov7153 — COMMENTED** [review]
Привет давай знакомиться я я Серёга

**Pull request merged**